pwiz_tools/Skyline/Model/DocSettings/SrmSettings.cs

```diff
@@ -1318,7 +1318,7 @@ public LibraryIonMobilityInfo GetIonMobilities(LibKey[] targetIons, MsDataFileUr
                     foreach (var ion in targetIons)

```

Comment:
I would have done:
foreach (var ion in targetIons.Distinct())
and skipped the dict.ContainsKey check below.